lib/sqlalchemy/sql/selectable.py

```diff
@@ -6706,7 +6747,7 @@ def correlate_except(
 
         e = self._clone()
         e.element = self._regroup(
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325